config/infection.json.dist

```diff
@@ -21,6 +21,48 @@
         }
     },
     "timeout": 10,
-    "minMsi": 98,
-    "minCoveredMsi": 100
+    "minMsi": 100,
+    "minCoveredMsi": 100,
+    "mutators": {
+        "@default": true,
+        "Break_": {
+            "ignore": [
+                "Psl\\Collection\\*Map::zip"
+            ]
+        },
+        "DecrementInteger": {
+            "ignore": [
+                "Psl\\DataStructure\\PriorityQueue::peek"
+            ]
+        },
+        "FunctionCallRemoval": {
+            "ignore": [
+                "Psl\\Result\\Success::getThrowable"
+            ]
+        },
+        "IncrementInteger": {
+            "ignore": [
+                "Psl\\DataStructure\\PriorityQueue::peek"
+            ]
+        },
+        "LogicalNot": {
+            "ignore": [
+                "Psl\\Hash\\Context::update"
+            ]
+        },
+        "MethodCallRemoval": {
+            "ignore": [
+                "Psl\\Iter\\Iterator::seek",
```

Comment:
These functions were added to speed up the tests : it results in time-outs cause the iterator does not iterate anymore because it removes:

https://github.com/azjezz/psl/blob/4affb62545080af0457f78a2bdcfc33cc4057f47/src/Psl/Iter/Iterator.php#L181